integration_tests/integration_test.ts

```diff
@@ -36,6 +36,16 @@ describe("test grpc-gateway-ts communication", () => {
     expect(response).to.deep.equal([2, 3, 4, 5, 6])
   })
 
+  it('binary echo', async () => {
+    const message = "ping";
+
+    const resp:any = await CounterService.EchoBinary({
+      data: new TextEncoder().encode(message),
+    }, { pathPrefix: "http://localhost:8081" })
+
+    expect(atob(resp["data"])).to.equal(message)
```

Comment:
still using `atob` here?

Reply:
yeah, as I couldn't figure out the flow to convert it back to `Uint8Array`. Didn't really want to expose base64 methods from fetch, but can do that instead.

Reply:
for the moment I think it is ok to do so. later on when the receiver side's been patched up we could hide it instead.